CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.0.13
+- Added the connectToExternalERs and limit parameters to ER lookup.

```

Comment:
I think we should avoid you using the ER abbreviation, "Event-Repository", since ER and Event-Repostory is not well known for external users and has not been open-sourced from Ericsson yet.
So I think we should write the full Event-Repository name in documantation and also in the variable names, if variable not becomes to long.
So we can maybe name the boolean variable name/parameter: "useExternalEventRepositories"

Reply:
Using EventRepository fullname instead of ER abbreviation comment applies to all places ER is used in this PR.
Thats my opinion.

Reply:
emili suggested to change the parameter names like below
connectToExternalERs to lookupInExternalERs
limit  to lookupLimit 
Done changes and pushed it, could you please check once and suggest it if its required

Reply:
ok.
We can maybe later create API version 2 with "v2/" as context-path pick better names in the new API version. Can maybe be a future task.